pod-operation/src/components/ina219.rs

```diff
@@ -0,0 +1,10 @@
+use ina219::INA219;
+use rppal::i2c::I2c;
+use tracing::debug;
+
+pub fn read_current(ina: &mut INA219<I2c>) -> f32 {
+	ina.calibrate(0xffff).unwrap();
+	debug!("Calibrating INA219");
+
+	ina.current().unwrap() as f32 / 160.0
```

Comment:
Chore: the calibration step should be needed only once per device. We could move this step to `PressureTransducer::new`.